hcloud/metrics/domain.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Dict, List, Literal, Tuple
+
+from dateutil.parser import isoparse
+
+from ..core import BaseDomain
+
+TimeSeries = Dict[str, Dict[Literal["values"], List[Tuple[float, str]]]]
+
+
+class Metrics(BaseDomain):
+    """Metrics Domain
+
+    :param start: Start of period of metrics reported.
+    :param end: End of period of metrics reported.
+    :param step: Resolution of results in seconds.
+    :param time_series: Dict with time series data, using the name of time series as key.
```

Comment:
Should we explain how this time series data is formatted inside the tuple? (`[ttimestamp, value]`)

Reply:
The full time series schema is overly complex, I would like to rely on the typing system to describe it. 

I could explain what the list of tuple are though!

Reply:
Thats what I meant, full thing is pretty complex, but I feel the example in the API docs is quite good to get a grasp.